index.html

```diff
@@ -3,8 +3,53 @@
 <head>
   <meta charset="UTF-8">
   <title>Team 2's projects</title>
+    <style>
+        .banner {
+            background-image: url("images/tech-wallpaper.png")
+        }
+    </style>
 </head>
 <body>
+ //  picture banner 
+ <section id="home" class="banner" style="height: 925px; margin-bottom: 50px;">
+      <div class="1-subsection">       
```

Comment:
I Think you're trying to apply a version of SMACSS? In which case should be `l-subsection`? Not really clear what's going on